### ISSUE TITLE
Any plans to add clustering to your compose?

### ISSUE THREAD

**Issue by jay-johnson** [LoyaltyNZ/alchemy-framework#2]
Not sure if you need clustering in the samples, but here's the one I built a while back:
https://github.com/GetLevvel/testing-rabbitmq-clustering-with-docker

Here's the compose:
https://github.com/GetLevvel/testing-rabbitmq-clustering-with-docker/blob/master/cluster/docker-compose.yml

**Comment by grahamjenson** [comment]
I don't think this will add much as it would just be clustering on a single machine. It is more interesting to have clustering on Vagrant, because then at least the machines are different (even if virtual).
